.sass-lint.yml

```diff
@@ -42,7 +43,7 @@ rules:
   variable-name-format:
     - 2
     -
-      convention: camelcase
+      convention: hyphenatedlowercase
```

Comment:
aqui não tem q ser `hyphenated_BEM`?

Reply:
Only for class name, for variables we should use hyphenatedlowercase